src/core/utils/eventutils.js

```diff
@@ -0,0 +1,40 @@
+import FilterType from '../filters/filtertype';
+import AnalyticsEvent from '../analytics/analyticsevent';
+import { AnswersAnalyticsError } from '../errors/errors';
+
+/**
+ * Create the 'REMOVED_FILTER' analytics event, using the given {@link FilterNode}.
+ * @param {Filter} removedFilter The {@link Filter} that was removed.
+ * @param {string} optionType The optionType to send in the request, either STATIC_FILTER, FACET_FILTER, or RADIUS_FILTER.
```

Comment:
The issue I was running into is that if a CombinedFilterNode is removed, then there is no FilterMetadata, which means there is no filterType since filterType lives on the FilterMetadata.

Reply:
removing analytics event so no longer an issue